src/main/java/com/documaster/rms/noark/ws/noarkentities/Dokumentversjon.java

```diff
@@ -29,6 +29,9 @@ public class Dokumentversjon extends NoarkEntityBase<Dokumentversjon> {
 	@JsonIgnore
 	private String filnavn;
 
+	@JsonIgnore
```

Comment:
I believe the field should be writeable too.

Reply:
The field is not writable, it's system managed as described in the documentation PR. I doublechecked by trying to set the field and the request failed for that reason.

Reply:
Ah, you are right! We should probably think about exposing it for writing in the future, but that wouldn't be a part of this PR then.

Thanks for the clarification.